src/azure-cli/azure/cli/command_modules/resource/tests/latest/deployment-scripts-deploy.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "scriptName": {
+           "type": "string",
+           "metadata": {
+                "description": "Deployment script name"
+            }
+        },
+        "utcValue": {
+            "type": "string",
+            "defaultValue": "[utcNow()]"
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Resources/deploymentScripts",
+            "apiVersion": "2019-10-01-preview",
+            "name": "[parameters('scriptName')]",
+            "location": "[resourceGroup().location]",
+            "identity": {
+                "type": "UserAssigned",
+                "userAssignedIdentities": {
+                    "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/python_sdk_test/providers/Microsoft.ManagedIdentity/userAssignedIdentities/python_test_sdk_msi')]": {}
```

Comment:
I suggest that this MSI can be generated dynamically in the test code, and then it's resourceId can be obtained and passed in via the `group deployment create xxx --parameters userAssignedIdentities={MSI_Resource_ID}` parameter.
This can avoid the problem that the rg and name of MSI are fixed but the MSI may not be built under the subscription used.